package.json

```diff
@@ -59,7 +60,6 @@
     "lint:styleguide": "eslint -c .eslintrc.styleguide.cjs src/",
     "lint:styleguide-fix": "eslint -c .eslintrc.styleguide.cjs src/ --fix",
     "prepare": "husky",
-    "pretest": "npm run typecheck && npm run lint",
```

Comment:
Can we remove the `npm run test -- --coverage` in the `build` stage then as well?

Reply:
Ok, it seems that my change was actually not needed. But this way it is clearer what happens. I remove the test run form the build script.